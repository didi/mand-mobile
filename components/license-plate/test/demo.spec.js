import Demo0 from './cases/demo0'
import {renderToString} from '@vue/server-test-utils'

describe('Landscape - Demo', () => {
  test('Basic', () => {
    expect(renderToString(Demo0)).toMatchSnapshot()
  })
})
