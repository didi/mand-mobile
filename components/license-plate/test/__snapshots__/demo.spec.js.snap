// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landscape - Demo Basic 1`] = `
<div>
  <div class="license-plate-container">
    <div>
      <div id="ac4c8e71-e294-736d-a047-8c2b04c44585_divisionInput" class="input-container division">
        <div class="input-widget">
          <div class="input-item active animation">
            <div class="input-item_content">

            </div>
          </div>
          <div class="input-item">
            <div class="input-item_content">

            </div>
          </div>
          <div class="input-item">
            <div class="input-item_content">

            </div>
          </div>
          <div class="input-item">
            <div class="input-item_content">

            </div>
          </div>
          <div class="input-item">
            <div class="input-item_content">

            </div>
          </div>
          <div class="input-item">
            <div class="input-item_content">

            </div>
          </div>
          <div class="input-item">
            <div class="input-item_content">

            </div>
          </div>
          <div class="input-item">
            <div class="input-item_content">
              <div class="emptyValue">
                <div>+</div>
                <div class="emptyValue_text">新能源</div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <!---->
    </div>
    <!---->
  </div>
</div>
`;
